_search-plugins/concurrent-segment-search.md

```diff
@@ -23,91 +20,26 @@ In OpenSearch, each search request follows the scatter-gather protocol. The coor
-
-If you are running an OpenSearch cluster and want to enable concurrent segment search in the config file, add the following line to `opensearch.yml`:
-
-```yaml
-opensearch.experimental.feature.concurrent_segment_search.enabled: true
-```
-{% include copy.html %}
-
-### Enable with Docker containers
-
-If you’re running Docker, add the following line to `docker-compose.yml` under the `opensearch-node` > `environment` section:
-
-```bash
-OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.concurrent_segment_search.enabled=true"
-```
-{% include copy.html %}
-
-### Enable on a node using a tarball installation
+## Enabling concurrent search at the index or cluster level
 
-To enable concurrent segment search on a tarball installation, provide the new JVM parameter either in `config/jvm.options` or `OPENSEARCH_JAVA_OPTS`.
-
-#### OPTION 1: Modify jvm.options
-
-Add the following lines to `config/jvm.options` before starting the `opensearch` process to enable the feature and its dependency:
-
-```bash
--Dopensearch.experimental.feature.concurrent_segment_search.enabled=true
-```
-{% include copy.html %}
-
-Then run OpenSearch:
-
-```bash
-./bin/opensearch
-```
-{% include copy.html %}
-
-#### OPTION 2: Enable with an environment variable
-
-As an alternative to directly modifying `config/jvm.options`, you can define the properties by using an environment variable. This can be done using a single command when you start OpenSearch or by defining the variable with `export`.
-
-To add these flags inline when starting OpenSearch, run the following command:
-
-```bash
-OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.concurrent_segment_search.enabled=true" ./opensearch-{{site.opensearch_version}}/bin/opensearch
-```
-{% include copy.html %}
-
-If you want to define the environment variable separately prior to running OpenSearch, run the following commands:
-
-```bash
-export OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.concurrent_segment_search.enabled=true"
-```
-{% include copy.html %}
-
-```bash
-./bin/opensearch
-```
-{% include copy.html %}
-
-## Disabling concurrent search at the index or cluster level
-
-After you enable the experimental feature flag, all search requests will use concurrent segment search during the query phase. To disable concurrent segment search for all indexes, set the following dynamic cluster setting:
+By default, concurrent segment search is disabled on the cluster. To enable concurrent segment search for all indexes in the cluster, set the following dynamic cluster setting:
 
 ```json
 PUT _cluster/settings
 {
    "persistent":{
-      "search.concurrent_segment_search.enabled": false
+      "search.concurrent_segment_search.enabled": true
    }
 }
 ```
 {% include copy-curl.html %}
 
-To disable concurrent segment search for a particular index, specify the index name in the endpoint:
+To enable concurrent segment search for a particular index, specify the index name in the endpoint:
```

Comment:
Can we mention that the index setting takes priority over the cluster setting? So if the cluster setting has concurrent search enabled but the index setting has it disabled, then it will be disabled for that index.